vtex-sites/gatsby.store
# Feat: Uses Hero component from FSUI

ported from https://github.com/vtex-sites/nextjs.store/pull/98

## What's the purpose of this pull request?

This PR is part of the Component Placement project. Moves `Hero` component to FSUI and replace it. 

![image](https://user-images.githubusercontent.com/3356699/172209135-8c2420cb-051d-4dc3-843e-90b5add4bc11.png)

## How does it work?

- Uses `Hero` component from FSUI and removes local one.
- Moves `hero.scss` to `src/components/sections/Hero/`
- Renames `data-fs-hero-content` token to `data-fs-hero-heading`

## How to test it?

Nothing should change visually on the component.

Check on the [preview](https://sfj-03cc55b--gatsby.preview.vtex.app/) if the Hero is working fine as [before](https://gatsby.vtex.app/).

## Checklist

<em>You may erase this after checking them all ;)</em>



- PR description
- [ ] For documentation changes, ping @ carolinamenezes, @ PedroAntunesCosta or @ Mariana-Caetano to review and update the changes.
